app/models/middleware_messaging.rb

```diff
@@ -0,0 +1,6 @@
+class MiddlewareMessaging < ApplicationRecord
+  belongs_to :ext_management_system, :foreign_key => "ems_id"
+  belongs_to :middleware_server, :foreign_key => "server_id"
```

Comment:
It would be duplicated when showing all messaging. However do we want to filter these out? What if some other middleware behaves differently? And in domain mode the destination of JMS would not be same for each server?